engine/z_versions_it_test.go

```diff
@@ -27,6 +27,7 @@ import (
 	"os/exec"
```

Comment:
Please add these tests in a separate PR from your strip so I can accept it.

Thanks!